Source/Csla.Blazor/State/NoOpSessionManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SessionServerManager.cs" company="Marimer LLC">
```

Comment:
The filename here is incorrect

Reply:
whoops - good catch!

OK I will do main branch now